rasa/core/policies/embedding_policy.py

```diff
@@ -137,31 +138,31 @@ def _standard_featurizer(cls):
             LabelTokenizerSingleStateFeaturizer())
 
     def __init__(
-        self,
-        featurizer: Optional[FullDialogueTrackerFeaturizer] = None,
-        priority: int = 1,
-        encoded_all_actions: Optional[np.ndarray] = None,
-        graph: Optional[tf.Graph] = None,
-        session: Optional[tf.Session] = None,
-        intent_placeholder: Optional[tf.Tensor] = None,
-        action_placeholder: Optional[tf.Tensor] = None,
-        slots_placeholder: Optional[tf.Tensor] = None,
-        prev_act_placeholder: Optional[tf.Tensor] = None,
-        dialogue_len: Optional[tf.Tensor] = None,
-        x_for_no_intent: Optional[tf.Tensor] = None,
-        y_for_no_action: Optional[tf.Tensor] = None,
-        y_for_action_listen: Optional[tf.Tensor] = None,
-        similarity_op: Optional[tf.Tensor] = None,
-        alignment_history: Optional[tf.Tensor] = None,
-        user_embed: Optional[tf.Tensor] = None,
-        bot_embed: Optional[tf.Tensor] = None,
-        slot_embed: Optional[tf.Tensor] = None,
-        dial_embed: Optional[tf.Tensor] = None,
-        rnn_embed: Optional[tf.Tensor] = None,
-        attn_embed: Optional[tf.Tensor] = None,
-        copy_attn_debug: Optional[tf.Tensor] = None,
-        all_time_masks: Optional[tf.Tensor] = None,
-        **kwargs: Any
+            self,
```

Comment:
Are these tabs or what are these changes?

Reply:
Yes. If I remove them, travis will fail due to wrong code style. Not sure what happened exactly. Sorry, for the many formatting issues.

Reply:
No, worries. Did you configure pycharm to use soft tabs?

Reply:
Yes.

Reply:
I checked it and indented it with 4 spaces (instead of 8) and `py.test --cache-clear --codestyle -m codestyle`did not complain . Also 8 spaces looks very weird to me. What would black do?

Reply:
black would indent it with 4 spaces

Reply:
Sorry, to bother you but could we change that back to 4 spaces now? Travis should definitely not complain about that unless its tabs or some weird stuff.

Reply:
I can try :)